app/models/theme_field.rb

```diff
@@ -4,6 +4,17 @@ class ThemeField < ActiveRecord::Base
 
   belongs_to :upload
 
+  scope :find_by_theme_ids, ->(theme_ids) {
```

Comment:
scopes have to be tested as well otherwise we wouldn't know if the query here is working or not.